bench/wtperf/wtperf_opt.i

```diff
@@ -80,6 +80,8 @@
  * The difference between CONFIG_STRING and STRING is that CONFIG_STRING
  * options are appended to existing content, whereas STRING options overwrite.
  */
+DEF_OPT_AS_UINT32(
+  backup_interval, 0, "backup the database every interval seconds during the workload phase, 0 to disable")
```

Comment:
This all looks fine. I think you should create a configuration file that shows it in use. I suggest adding a new file based off of `wtperf/runners/multi-btree.wtperf` (or one of its variants if you want backup with other things too) since that uses multiple tables.

Reply:
I tried it with one of the configuration file and it is working. There are following tickets to add backup to the configs.